conf/application.conf

```diff
@@ -21,6 +21,7 @@ appName = economic-crime-levy-account
 
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.backend.BackendModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
```

Comment:
If we are no longer using V1, we should remove it.

Reply:
Removing it caused all the unit test suite to fail as it couldn't locate HttpModule for the DefaultAuthConnector? Can look into it though as there probably is a solution for it

Reply:
That's fine. Sounds like a hard dependency.